docs/functions/ecg.rst

```diff
@@ -71,7 +71,9 @@ Miscellaneous
 """"""""""""""""""""""""
 .. autofunction:: neurokit2.ecg.ecg_findpeaks
 
-
+*ecg_invert()*
+""""""""""""""""""""""""
+.. autofunction:: neurokit2.ecg.ecg_invert
 
 *Any function appearing below this point is not explicitly part of the documentation and should be added. Please open an issue if there is one.*
 
```

Comment:
we should also add any new function to the exclude-members below, basically the section below documents all the functions from - in this case - the ecg module with the exceptions of the ones I specified: this was used as a check for me to make sure I didn't forget any functions but if we don't add new functions to the exclusion it will be documented two times